smpte.js

```diff
@@ -126,25 +126,22 @@ function insertTOC(docMetadata) {
       if (subSection.localName !== "section")
         continue;
 
-      if (!subSection.hasAttribute("id"))
+      const secId = subSection.getAttribute("id")
+
+      if (!secId || ['sec-front-matter', 'sec-toc'].includes(secId))
         continue;
 
       const heading = subSection.firstElementChild;
 
       if (!heading)
         continue;
 
-      const headingNumber = subSection.querySelector(".heading-number");
-
-      if (! headingNumber)
-        continue;
-
       subSections.push(subSection);
     }
 
     if (subSections.length > 0) {
 
-      const tocItem = document.createElement("ol");
+      const tocItem = document.createElement("ul");
```

Comment:
The ToC is an ordered list, right?

Reply:
Resolved in https://github.com/SMPTE/html-pub/pull/27/commits/164cbd6c669f72883acd258fbb2d8809377d5686